apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,389 @@
+            Manage Access
+          </button>
+          <dialog bind:this={fileAccessModal} class="modal modal-bottom sm:modal-middle">
+            <div class="modal-box">
+              <h3 class="font-bold text-lg pb-2">Share Examplefile.ppt</h3>
+              <input
+                type="text"
+                placeholder="Add People or Groups"
+                class="
+                  input input-bordered input-info w-full text-sm
+                  focus:ring-info focus:border-info focus:outline-none" />
+              <p class="py-4 font-semibold">People with Access</p>
+
+              <!-- Users with Access -->
+              {#await getUserDetails() then userDetails}
+                {#if userDetails}
+                  <div class="flex items-center py-2">
+                    <img class="rounded w-9 mr-3" alt="Error" {src} />
+                    <div class="flex flex-col">
+                      <div class="font-bold">{userDetails.full_name} (You)</div>
+                      <div class="text-sm">{userDetails.email}</div>
+                    </div>
+                  </div>
+                {/if}
+              {/await}
+
+              <div class="modal-action flex justify-between">
+                <!-- Copy Link Button -->
+                <div class="flex justify-start">
+                  <button
+                    class="
+                      btn btn-outline btn-info rounded-full
+                      font-semibold normal-case px-6 flex items-center">
+                    <img src="linkIcon.svg" alt="Error" />
+                    Copy Link
+                  </button>
+                </div>
+
+                <form method="dialog">
+                  <!-- if there is a button in form, it will close the modal -->
+                  <button class="btn">Done</button>
+                </form>
+              </div>
+            </div>
+          </dialog>
+        </div>
+
+        <!-- File Type Section -->
+        <div class="py-2 text-sm">
+          <p>Type</p>
+          <p class="text-info">PowerPoint</p>
+        </div>
+
+        <!-- File Size Section -->
+        <div class="py-2 text-sm">
+          <p>Size</p>
+          <p class="text-info">57.3MB</p>
+        </div>
+
+        <!-- Manual Self Destruct Button -->
+        <div class="pt-4">
+          <div class="py-2">
+            <button class="py-4 w-full btn" on:click={showSelfDestructModal}>
+              SELF-DESTRUCT
+            </button>
+            <dialog bind:this={selfDestructModal} class="modal modal-bottom sm:modal-middle">
+              <div class="modal-box">
+                <h3 class="font-bold text-lg">Warning</h3>
+                <p class="py-4">
+                  This will
+                  <span class="text-error"> destroy all active instances of this file </span>
+                  , are you sure?
+                </p>
+                <button
+                  class="
+                    btn btn-outline btn-error px-8">
```

Comment:
If these 2 lines can be combined into one, why not make it one?

Reply:
Done